pom.xml

```diff
@@ -24,8 +24,9 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <jandex-plugin.version>1.2.3</jandex-plugin.version>
-        <quarkus.version>2.16.3.Final</quarkus.version>
+        <quarkus.version>3.0.0.Alpha4</quarkus.version>
```

Comment:
Can you revert here please?

Reply:
But that's the goal of the PR.

Reply:
Yeah, but the bot will update it as soon as we do the pre-release PR.

Reply:
duiahsduihaskjdaskdasdhasklj forget it. And thanks for the heads up in the chat. This is the quarkus version. 🤦